src/config/mod.rs

```diff
@@ -78,6 +78,9 @@ pub struct Config {
     // Enable/disable notification replace functionality.  I don't like how some apps do it.
     #[serde(default = "maths_utility::val_true")]
     pub replacing_enabled: bool,
+    // Whether or not to refresh the timeout of a notification on an update
+    #[serde(default = "maths_utility::val_false")]
+    pub replace_refresh_timeout_enabled: bool,
```

Comment:
Sorry to be pedantic, but can we name this `replacing_resets_timeout`?

```suggestion
    pub replacing_resets_timeout: bool,
```